src/constructs/sns/sns-topic.ts

```diff
@@ -21,7 +20,7 @@ interface GuSnsTopicProps extends TopicProps, GuMigratingResource {}
  *  new GuSnsTopic(stack, "SnsTopic", { existingLogicalId: "MyCloudFormedSnsTopic" });
  * ```
  */
-export class GuSnsTopic extends GuStatefulMigratableConstruct(Topic) {
+export class GuSnsTopic extends Topic {
```

Comment:
This construct is now a pure alias to the AWS one and isn't needed. I'd suggest we remove it, but not sure if it's correct to do here or in it's own PR?